moveit_core/urdfdom_compatibility.h.in

```diff
@@ -0,0 +1,78 @@
+/*********************************************************************
```

Comment:
Does not seem like this belongs in the root folder of ``moveit_core``, how about ``moveit_core/macros/include/moveit/macros/`` ?